cartographer/mapping/internal/2d/pose_graph_2d.h

```diff
@@ -257,6 +258,9 @@ class PoseGraph2D : public PoseGraph {
   // Set of all frozen trajectories not being optimized.
   std::set<int> frozen_trajectories_ GUARDED_BY(mutex_);
 
+  // Whether or not optimize landmark poses.
+  bool freeze_landmarks_ GUARDED_BY(mutex_) = false;
```

Comment:
IIRC default values are to be avoided, according to the Google coding style.

Reply:
Style guide talks mostly about [default arguments](https://google.github.io/styleguide/cppguide.html#Default_Arguments) in functions. They are allowed now (it was changed a 1+ year ago).